File: msgraph-mail/typescript/src/users/inferenceClassificationType.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class InferenceClassificationType implements Parsable<InferenceClassificationType> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: InferenceClassificationType, node: ParseNode) => void> {
        return new Map<string, (item: InferenceClassificationType, node: ParseNode) => void>([
        ]);
    };
}

File: msgraph-mail/dotnet/Users/InferenceClassificationType.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users {
    public class InferenceClassificationType : IParsable<InferenceClassificationType> {
        public IDictionary<string, Action<InferenceClassificationType, IParseNode>> DeserializeFields => new Dictionary<string, Action<InferenceClassificationType, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}
